projects/BFDR/IJEFetalDeath.cs

```diff
@@ -71,6 +71,46 @@ public new FetalDeathRecord ToRecord()
             return this.record;
         }
 
+        /// <summary>Converts the FHIR representation of presence-only fields to the IJE equivalent.</summary>
+        /// <param name="fieldValue">the value of the field</param>
+        /// <param name="noneOfTheAboveValue">the value of the corresponding none-of-the-above field</param>
+        /// <returns>Y (yes), N (no), or U (unknown)</returns>
+        private string PresenceToIJE(bool fieldValue, bool noneOfTheAboveValue)
```

Comment:
Are these fields the same as in IJEBirth.cs? If both files use them, they should be moved to VitalRecord/IJE.cs instead so we just define them once.